shell/assets/translations/en-us.yaml

```diff
@@ -4587,7 +4587,7 @@ resourceTable:
     machinePool: "<span>Pool:</span> {name}"
     notInANamespace: Not Namespaced
     notInAProject: Not in a Project
-    project: "<span>Project:</span> {name}"
+    project: "Project:"
```

Comment:
The name param is here so that for rtl locales, - can't always assume that the name should go after the label.